### PR TITLE
Dockerfile and script to build a standalone version in

### PR DESCRIPTION
**Pull request by dig412**
I wanted to build a 64bit version for XenServer 7, so I made this Dockerfile and script to do it in a repeatable way.

### CODE REVIEW

**Comment by dig412** [comment]
Thanks @djamps! 
I've built a 64bit binary and made it available on my fork's releases page, if you'd like to copy it to your repo:
https://github.com/dig412/migrate-vm/releases

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,10 @@
+FROM centos:centos7
```
Comment (bharsh) [inline Dockerfile:1]: Hi - thanks to both @dig412 and @djamps for keeping this going - I had wanted an updated build for XenServer 6.5 (I know, it's old, but it has free features I like and we don't want to be forced into a paid plan) - I'm trying this now, but is it as simple as changing the from: line to "centos6" and having it build for CentOS6 instead?!  Here's hoping... hm.  almost, but it complains about "cpanm - command not found"  I'll keep trying things, but if either of you know how to best do this, that would be awesome...

Comment (bharsh) [reply Dockerfile:1]: hasn't quite worked.  I think I figured out how to add a line to pull in cpanminus in c6 but still seem to be in dependency hell due to ssl stuff now.  I'll keep digging at it but if either of you are really expert at this, let me know how to proceed.  I'm just in try-fail-repeat mode now - first time ever with docker so it's a curve.  If I get it, I'll send a pull request.

Comment (dig412) [reply Dockerfile:1]: @bharsh 
I've had a muck around with this and managed to get it to build with a couple of extra packages from a different repo.
Could you try checking out my branch at https://github.com/dig412/migrate-vm/tree/docker-build-xen6 and seeing if that works for you?
You can run the new script `docker_build_xen6.sh` to do a full build and extract the tool.

**Pull request merged**